src/generators/mappersGenerator.ts

```diff
@@ -105,7 +106,9 @@ function writeMapper(writer: CodeBlockWriter, mapper: Mapper) {
   } = type as CompositeMapperType;
   const isReferenceDicriminator = isString(polymorphicDiscriminator);
   writer.block(() => {
-    writeObjectProps(restMapper, writer)
+    const { defaultValue, ...mapperProps } = restMapper;
```

Comment:
Some of these sections are starting to have enough code that it'd be helpful to start inserting comments here and there explaining what the different sections do.  Maybe even some simple example code in comments to show what the output looks like?

This isn't super important for now but it'll be useful in the long term when different people are trying to maintain this code.